src/ansys/tools/visualization_interface/backends/pyvista/pyvista.py

```diff
@@ -59,12 +58,6 @@
 from ansys.tools.visualization_interface.utils.logger import logger
 
 
-class PickingMode(Enum):
```

Comment:
Why deleting it?? If both modes are incompatible, how are you going to check which one is active?

Reply:
I find using two opposite flags a bit confused. But I would follow @RobPasMue and @ahernsean advice.

If one is going to take precedence, the code should raise an exception if both are active ``Explicit is better than implicit.`` [1](https://peps.python.org/pep-0020/).

Reply:
I raise a warning in that case, I don't think having both flags active should stop the program execution.